### PR TITLE
Add Asset Conversion Guides

### PR DESCRIPTION
**Pull request by DrW3RK**
- [x] Polkadot-JS guides demonstrating Asset Conversion Pallet functionalities
- ~~[ ] User Journeys - Parachains~~ PR Already big. Will do it through a followup PR.

### CODE REVIEW

File: docs/learn/learn-asset-conversion-assethub.md [hunk 1]
```diff
@@ -41,23 +41,24 @@ Asset Conversion pallet allows you to:
 ## Fee Payment in Any Asset - Wallets
 
 Without Asset Conversion pallet, only DOT and sufficient assets can be used for paying transaction
```
Comment (CrackTheCode016) [inline docs/learn/learn-asset-conversion-assethub.md]:
```suggestion
Without Asset Conversion pallet, only DOT and [sufficient assets](./learn-assets.md#sufficient-assets) can be used for paying transaction
```

**Review by CrackTheCode016 — COMMENTED** [review]
Overall looks good to me, that UI is really nice! 

Should the bottom half of this page (the guide essentially) be its own page that exists under PJS Guides, and the user journey placed in this page instead?

File: docs/learn/learn-asset-conversion-assethub.md [hunk 2]
```diff
@@ -67,18 +68,17 @@ control them.
...
-  without any infrastructure lift.
+broadcast transactions on each parachain. These parachains could use different balances pallets or
+order them differently. When Polkadot hosts hundreds of parachains (and possibly thousands of
+threads), this becomes a huge burden in terms of network support. This is in contrast to adding
+support for an additional ERC20 token; the marginal effort is very small since an Ethereum node
+already serves all the data they need.
+
+With support for these assets on AssetHub, exchanges/custodians could monitor one chain for deposits
+(applications and UIs would need to give the option to transfer to AssetHub). For withdrawals, users
+could choose to withdraw to their address on AssetHub. There are two user experience bonuses here:
```
Comment (joepetrowski) [inline docs/learn/learn-asset-conversion-assethub.md]:
```suggestion
With support for these assets on Asset Hub, exchanges/custodians could monitor one chain for deposits
(applications and UIs would need to give the option to transfer to Asset Hub). For withdrawals, users
could choose to withdraw to their address on Asset Hub. There are two user experience bonuses here:
```

File: docs/learn/learn-guides-asset-conversion.md [hunk 3]
```diff
@@ -17,12 +17,50 @@ see the [wallets](./wallets-index), [apps](./apps-index) and [dashboard](./dashb
 
 </div>
 
+:::info Wallet and UI Developers
+
+The [DOT ACP UI](https://github.com/MVPWorkshop/dot-acp-ui) project is part of Polkadot initiative
+for building front-end and UI for Asset Conversion Pallet on Polkadot's AssetHub. Currently, the app
```
Comment (joepetrowski) [inline docs/learn/learn-guides-asset-conversion.md]:
```suggestion
for building front-end and UI for Asset Conversion Pallet on Polkadot's Asset Hub. Currently, the app
```

File: docs/learn/learn-guides-asset-conversion.md [hunk 4]
```diff
@@ -17,12 +17,50 @@ see the [wallets](./wallets-index), [apps](./apps-index) and [dashboard](./dashb
...
+liquidity pool. If you are looking for a guide on how to create an asset on Asset Hub, it is
+available [here.](./learn-guides-assets-create.md)
+
 The `create_pool` function is used to create an empty liquidity pool along with a new `lp_token`
 asset. This asset's ID is announced in the `Event::PoolCreated` event. After creating a pool,
 liquidity can be added to it via the `Pallet::add_liquidity` function.
 
+For example, the snapshot below shows how to create liquidity pool with `ROC` tokens and `EDU`
+tokens with the asset ID `149` on Rococo Asset Hub. Creating this extrinsic requires knowledge of
+[XCM Multilocations](xcm/fundamentals/multilocation/summary.md). From the perspective of AssetHub,
```
Comment (joepetrowski) [inline docs/learn/learn-guides-asset-conversion.md:43]: These are now `Location`s as of XCM v4. May need a new link?

**Review by joepetrowski — APPROVED** [review]
Looks good. I think we should make a list of top asset-related user stories for parachain and wallet developers. So we could also have guides on:

1. How to create and manage a foreign asset
2. How to pay fees using assets

With Rust/TS examples for both.

File: docs/learn/learn-guides-asset-conversion.md (hunk 4 above)
Comment (CrackTheCode016) [reply docs/learn/learn-guides-asset-conversion.md:43]: I can take a look at the XCM docs tomorrow and see where they may need changes for v4, cc @DrW3RK

**Review by CrackTheCode016 — APPROVED** [review]

File: docs/learn/learn-guides-asset-conversion.md (hunk 4 above)
Comment (profradha) [reply docs/learn/learn-guides-asset-conversion.md:43]: Thanks. I created this issue to track what was requested here https://github.com/w3f/polkadot-wiki/issues/5830

**Pull request merged**